source/heroes/ScreensaverHero.hx

```diff
@@ -5,43 +5,67 @@ import flixel.util.FlxColor;
 import flixel.FlxG;
 
 class ScreensaverHero extends FlxSprite {
-    var DirectionX:Float;
-    var DirectionY:Float;
+    var HeroHeight:Int = 50;
+    var HeroWidth:Int = 25;
+    var StartingHeroColor:FlxColor = 0xFFFFFFFF;
+    var Blue:FlxColor = 0xFF0000FF;
+    var Yellow:FlxColor = 0xFFFFFF00;
+    var Green:FlxColor = 0xFF008000;
+    var Red:FlxColor = 0xFFFF0000;
+    var SpeedX:Float = FlxG.random.float(-50, 50);
+    var SpeedY:Float = FlxG.random.float(-50, 50);
 
     public function new(X:Float=0, Y:Float=0)
     {
         super(X, Y);
-        makeGraphic(20, 40, 0xFFFFFFFF);
-        this.DirectionX = FlxG.random.float(0,1);
-        this.DirectionY = FlxG.random.float(0,1);
+        super.makeGraphic(HeroWidth, HeroHeight, StartingHeroColor);
+        this.velocity.x = SpeedX;
+        this.velocity.y = SpeedY;
     }
     override public function update(elapsed:Float)
```

Comment:
This update function has gotten several different distinct kinds of functionality in it. The fact that those comments are useful ways to determine what's happening in each part of the `update()` method is a pretty clear sign of that.

Rather than using comments, I recommend breaking apart our `update()`s mega-functionality into several private helper methods. Then our `update()` will be very readable and we can dive in (and adjust, if needed) the functionality of each distinct chunk of code without risk of damaging the others.

In this case, I think we can safely say that you've got the following tasks being taken care of in here:
* screen wrap
* accelerate / change velocity in response to input
* reset speed in response to input
* change color based on screen position

So each of those might make sense to break out into their own method.

No required action here, but I think it'd be good practice to make things cleaner.